src/main/java/uk/gov/companieshouse/api/accounts/model/entity/BaseEntity.java

```diff
@@ -0,0 +1,5 @@
+package uk.gov.companieshouse.api.accounts.model.entity;
+
+public interface BaseEntity {
+
```

Comment:
I believe we will have to add `ID` into this interface?

Reply:
I wasn't sure that the ID implementation was set in stone, happy to add it if it is.

Reply:
Yeah, its set in stone,  we just need to implement it now.

Reply:
Done.